v0/destinations/appcues/transform.js

```diff
@@ -0,0 +1,184 @@
+const get = require("get-value");
+const set = require("set-value");
+const {
+  EventType
+} = require("../../../constants");
+
+const {
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  getValueFromMessage,
+  removeUndefinedAndNullValues,
+  removeUndefinedAndNullAndEmptyValues,
+} = require("../../util");
+
+const {
+  ConfigCategory,
+  mappingConfig,
+  getEndpoint,
+} = require("./config");
+
+const {
+  utcTimeToEpoch,
+} = require("./utils");
+
+function buildResponse(message, payload, endpoint) {
+  const response = defaultRequestConfig();
+  response.endpoint = endpoint;
+  response.userId = message.userId || message.anonymousId;
```

Comment:
Remove this. We don't use this `userId` any more.

Reply:
removed